requirements/collect_env_details.py

```diff
@@ -25,6 +25,7 @@
 import tqdm
 
 sys.path += [os.path.abspath(".."), os.path.abspath("")]
+import lightning_app  # noqa: E402
```

Comment:
This is done in https://github.com/Lightning-AI/lightning/pull/13815 already

Reply:
nice !